### ISSUE TITLE
[ cashier ][ error ] Sending empty `tax_percent` causing 400 fail

### ISSUE THREAD

**Issue by Kryptonit3** [laravel/framework#8645]
Not sure what is going on, I have not set tax_percent but it is apparently being sent and because it is empty ( NULL ) the Stripe API is throwing an error. Here is a more detailed post.

Invalid decimal: ; must contain at maximum two decimal places.

https://laracasts.com/discuss/channels/general-discussion/stripe-cashier-error-invalid-decimal-must-contain-at-maximum-two-decimal-places

**Comment by Kryptonit3** [comment]
I was able to circumvent this error by adding a `getTaxPercent()` method to my user model with a value of `0` but I was under the impression that it was optional.

**Comment by jpavon** [comment]
I'm having this same error when resuming a subscription, doing:

```php
$this->currentUser->subscription('pro')->resume($request->stripeToken);
```
The solution @Kryptonit3 suggested fixes the problem for me also... Ping @taylorotwell

**Comment by taylorotwell** [comment]
Fixed 5.0.4.
